### PR TITLE
Add payment method

### PR DESCRIPTION
**Pull request by rebeccafae**
[#97594338]

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -61,6 +61,3 @@ PLATFORMS
 
 DEPENDENCIES
   guard-rubocop
-
-BUNDLED WITH
```
Comment (jasonnoble) [inline Gemfile.lock:65]: Why was this removed?

Comment (rebeccafae) [inline Gemfile.lock:65]: Not sure... I never even went into this file (or at least I didn't think I did)
